data/styles/clear/include/Basemap.mapcss

```diff
@@ -569,6 +572,10 @@ area|z15-19[aeroway=aerodrome]
 area|z15-[landuse=churchyard]
 {fill-color: @background;}
 
+/* 7.8 Agricultural */
+area|z10-[landuse=farmland],
+{fill-color: @farmland;fill-opacity: 1;}
```

Comment:
Let's keep consistency here and draw farmlands starting from z12 like other similar landuses?

Reply:
My thinking was to show them at a higher zoom level to give a better idea of land use similar to how we do with forests. I would call farmland more similar to forests than something like hospitals or fields; but I see the argument for either.